backend/routes/postReaction.py

```
@@ -79,6 +81,23 @@ def like_action(like_data: LikeCreate, db: Session = Depends(get_db), current_us
         }
 
     new_like = add_like(like_data, db, current_user)
+
+    if like_data.post_id:
+        post = db.query(Post).filter(Post.id == like_data.post_id).first()
+        if post and post.user_id != current_user.id:  # Don't notify if liking own post
+            notification_data = NotificationCreate(
```

Ruff (0.8.2)

Local variable `notification_data` is assigned to but never used

Remove assignment to unused variable `notification_data`

(F841)